GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -2733,16 +2735,15 @@ private void DiffContextMenu_Opening(object sender, System.ComponentModel.Cancel
                 artificialRevSelected = artificialRevSelected || selectedRevisions[selectedRevisions.Count - 1].IsArtificial();
 
             // disable items that need exactly one selected item
-            bool isExcactlyOneItemSelected = DiffFiles.SelectedItems.Count() == 1;
-            var isCombinedDiff = isExcactlyOneItemSelected &&
+            bool isExactlyOneItemSelected = DiffFiles.SelectedItems.Count() == 1;
+            var isCombinedDiff = isExactlyOneItemSelected &&
                 DiffFiles.CombinedDiff.Text == DiffFiles.SelectedItemParent;
-            var enabled = isExcactlyOneItemSelected && !isCombinedDiff;
```

Comment:
Yes, I removed it intentionally. Formerly, it would disable opening with difftool is multiple files were selected. My change is to allow multiple file selection to open with difftool. By default, Enabled is true, so by removing this line, it's always enabled.